graphics/hugin-app/Portfile

```diff
@@ -33,7 +33,12 @@ use_bzip2               yes
 compiler.cxx_standard   2011
 
 perl5.branches          5.34
-wxWidgets.use           wxWidgets-3.0-cxx11
+# use wxWidget-3.2 on macOS Mojave (10.14) and later for dark mode support
+if {${os.major} > 13 || (${os.major} == 10 && ${os.minor} >= 14)} {
```

Comment:
this check is not correct. It should first check whether it's being used on macOS and then `os.major` refers to the Darwin version ([see this link](https://trac.macports.org/wiki/XcodeVersionInfo#macOSVersionMap)).

So it should read: ```if {${os.platform} eq "darwin" && ${os.major} >= 18} {```.

Once you make that change we'll merge.

Reply:
Thanks for the hint. I now adapted the portfile accordingly.